tests/tests/authority_tests.cpp

```diff
@@ -1858,43 +1879,129 @@ BOOST_FIXTURE_TEST_CASE( parent_owner_test, database_fixture )
    }
 }
 
-BOOST_FIXTURE_TEST_CASE( owner_delegation_test, database_fixture )
```

Comment:
This test case (`owner_delegation_test`) was accidentally removed in commit https://github.com/bitshares/bitshares-core/commit/771e03806d2f83c6ee7ba4a72ccd599decd4e31b. I think we should re-add it.